### ISSUE TITLE
<meta> element without "property" should be reported as error

### ISSUE THREAD

**Issue by babatakao** [w3c/epubcheck#1269]
Package document spec (both http://idpf.org/epub/31/spec/epub-packages.html#sec-meta-elem and https://www.w3.org/TR/epub-33/#sec-meta-elem) says that `<meta>` element in OPF must have `property` attribute.

> property [required]

However, current version of epubcheck does not report errors about it.

```xml
<package
 xmlns="http://www.idpf.org/2007/opf"
 version="3.0"
 unique-identifier="unique-id"
>

<metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
  <dc:title>Test</dc:title>

  <!-- Valid -->
  <meta property="dcterms:modified">2012-10-01T00:00:00Z</meta>

  <!-- Invalid -->
  <meta name="test" content="test"/>

  ...
</metadata>
<manifest>
  ...
```

In this case the second `meta` element should be reported as an error. (and it'll be better to reported warnings about non-standard attribute `name` and `content`. )

Note that, unfortunately `meta` elements without `property` are already widely used (e.g. Kindle https://kindlegen.s3.amazonaws.com/AmazonKindlePublishingGuidelines.pdf requests `<meta name="book-type">`) so the fix will affect to numerous contents.

**Comment by rdeltour** [comment]
Thanks for the report @babatakao!

But this is not a bug: in fact there are two different `meta` element:
- the regular EPUB 3 `meta` element which indeed requires a `property` attribute
- and also the **legacy `meta` element** which is defined in [OPF 2.0.1](http://idpf.org/epub/20/spec/OPF_2.0.1_draft.htm#Section2.2). This old `meta` element is still [allowed in EPUB 3](https://www.w3.org/publishing/epub32/epub-packages.html#sec-opf2-meta).

I'm closing this issue as `invalid` (not a bug), but feel free to keep on commenting here! 😊

**Comment by babatakao** [comment]
Thank you for the explanation! I've understood.
